### PR TITLE
[EL] Added HassShoppingListAddItem 

### PR DESCRIPTION
**Pull request by apo-mak**
Added HassShoppingListAddItem

### CODE REVIEW

File: sentences/el/shopping_list_HassShoppingListAddItem.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: "el"
+intents:
+  HassShoppingListAddItem:
+    data:
+      - sentences:
+          - πρόσθεσε <item> (στο | στη | στην )<my_list>
```
Comment (dsimop) [inline sentences/el/shopping_list_HassShoppingListAddItem.yaml:6]: - πρόσθεσε <item> (στο | στη | στην )<my_list> [μου]

File: sentences/el/shopping_list_HassShoppingListAddItem.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+          - βάλε <item> (στο | στη | στην )<my_list>
```
Comment (dsimop) [inline sentences/el/shopping_list_HassShoppingListAddItem.yaml:7]: - βάλε <item> (στο | στη | στην) <my_list> [μου]

**Review by dsimop — CHANGES_REQUESTED** [review]
Thank you for your PRs. Please consider the commented recommendations.

**Comment by dsimop** [comment]
The Build Pipeline failed. Please check the output in order to fix it. Thank you very much fr your contributions.

**Review by dsimop — CHANGES_REQUESTED** [review]
The Build Pipeline failed. Please check the output in order to fix it. Thank you very much for your contributions.

**Comment by apo-mak** [comment]
> The Build Pipeline failed. Please check the output in order to fix it. Thank you very much for your contributions.
it was probably missing the inline <item>.

**Review by dsimop — CHANGES_REQUESTED** [review]
The tests have to pass in order the PR to be merged.

**Comment by apo-mak** [comment]
new build Error:
`FAILED tests/test_language_sentences.py::test_shopping_list_HassShoppingListAddItem[el] - hassil.recognize.MissingListError: Missing slot list {shopping_list_item}`

Not sure why ... the test file is now indentical to the EN and DE one.

**Review by dsimop — CHANGES_REQUESTED** [review]
You need to add to the sentences/el/_common.yaml after line 293, the following code block:
```
  shopping_list_item:
    wildcard: true
```

You can check anytime if the tests are successful before committing and pushing with the following command on the terminal:
`pytest tests --language el`

**Comment by apo-mak** [comment]
> pytest tests --language el

now it is fixed 

`================================================================================================================================= test session starts ==================================================================================================================================
platform linux -- Python 3.10.12, pytest-7.3.1, pluggy-1.0.0
rootdir: /workspaces/intents
configfile: pyproject.toml
collected 45 items                                                                                                                                                                                                                                                                     

tests/test_language_intents.py .......................                                                                                                                                                                                                                           [ 51%]
tests/test_language_sentences.py ......................                                                                                                                                                                                                                          [100%]

================================================================================================================================== 45 passed in 4.39s ==================================================================================================================================`

thank you for pointing this out.

**Review by dsimop — APPROVED** [review]
Thank you very much again for your contributions and your time. 👍

**Pull request merged**